Community/README.md

```diff
@@ -0,0 +1,48 @@
+# Community
+
+Entrypoint for getting onboarded.
+
+## Tools used by the community
+
+- Instant messaging
+  - [Matrix](https://matrix.to/#/!TiDqlLmEUaXqTemaLc:matrix.org?via=matrix.org)
+  - [Nextcloud Talk](https://scs.sovereignit.de/nextcloud) (Not used)
```

Comment:
If Talk is not used, we shouldn't list it here either, or should we?

Reply:
Depends, technically, we have the tool available. Not sure.

Reply:
IMHO, we should focus on matrix as gaia-x uses it as well.